core/jazz_splunk-kinesis-log-streamer/index.js

```diff
@@ -36,8 +36,9 @@ function handler(event, context, callback) {
   logger.debug('Received event:' + JSON.stringify(event));
```

Comment:
For some reason, you are missing `logger.init` before using `logger`
```suggestion
  logger.init (event, context));
  logger.debug('Received event: ' + JSON.stringify(event));
```

Reply:
Included